Snow-Media-Technologies/filmo-backend
# chore/filmo 0023 generate modal and migration files

## Description
- Create required model and migration files for all models of the app

## Type of change

## How Has This Been Tested?
Please describe the tests that you ran to verify your change

## How to Manually Test
- Ensure that you have postgres database installed on your local machine
- Checkout to this branch
- Create a file and name it `.env`
- Add the database url as shown below
`DATABASE_URL='postgres://postgres:database_password@127.0.0.1:5432/filmo'`
- Run the command `npm run migrate` to create the database table
- The command above should create tables within the database

## Checklist:

## Design prototype link(s) (for UI stories)
- N/A

## Implementation Plan link(s)

## Screenshots

## JIRA
[Filmo-0023](https://filmo.atlassian.net/browse/FB-23)
